src/GovUk.Education.ExploreEducationStatistics.Publisher.Model/ReleaseInfo.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Microsoft.Azure.Cosmos.Table;
+
+namespace GovUk.Education.ExploreEducationStatistics.Publisher.Model
+{
+    public class ReleaseInfo : TableEntity
```

Comment:
Might be worth putting a comment here describing the purpose of ReleaseInfo (or renaming it), as it wasn't immediately clear to me what it represented based on the name.

Reply:
I've renamed it to ReleaseStatus now, hopefully that is a bit clearer.